include/LMS1xx/MRS1000.h

```diff
@@ -1,16 +1,33 @@
 #ifndef MRS1000_H
 #define MRS1000_H
-#include <LMS1xx/LMS1xx.h>
+#include <LMS1xx/colaa.h>
 
-class MRS1000 : public LMS1xx
+struct dataChannel
+{
+  int channel_nr;
+  uint16_t data_len;
+  uint16_t dist[1101];
+  uint16_t rssi[1101];
+};
+
+struct scanDataLayerMRS
+{
+  bool first;
+  int scan_nr;
+  uint32_t time_since_startup;
+  uint32_t transmission_duration;
+  double layer_angle;
+  int layer_nr; // starting from 0;
+  dataChannel channel[3];
+};
+
+
+class MRS1000 : public CoLaA
 {
 public:
-  bool getScanData(scanDataLayerMRS* scan_data);
-  bool getScanDataLMSProtocol(scanDataLayerMRS *scan_data);
 
 protected:
-  static bool parseScanLayer(char* buffer, scanDataLayerMRS* scan_data);
-  static bool parseScanLayerLMSProtocol(char *buffer, scanDataLayerMRS *scan_data);
+  void parse_scan_data(char *buffer, void *scan_data);
```

Comment:
[Style Guide](http://wiki.ros.org/CppStyleGuide) says functions should be **camelCased**